acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/transaction/IbftTwoJsonRpcRequestFactory.java

```diff
@@ -26,41 +26,41 @@
 import org.web3j.protocol.Web3jService;
 import org.web3j.protocol.core.Request;
 
-public class IbftJsonRpcRequestFactory {
+public class IbftTwoJsonRpcRequestFactory {
 
   private final Web3jService web3jService;
 
-  public IbftJsonRpcRequestFactory(final Web3jService web3jService) {
+  public IbftTwoJsonRpcRequestFactory(final Web3jService web3jService) {
     this.web3jService = web3jService;
   }
 
-  public Request<?, ProposeResponse> ibftPropose(final String address, final Boolean auth) {
+  public Request<?, ProposeResponse> propose(final String address, final Boolean auth) {
     return new Request<>(
         "ibft_proposeValidatorVote",
         Arrays.asList(address, auth.toString()),
         web3jService,
         ProposeResponse.class);
   }
 
-  public Request<?, DiscardResponse> ibftDiscard(final String address) {
+  public Request<?, DiscardResponse> discard(final String address) {
     return new Request<>(
         "ibft_discardValidatorVote", singletonList(address), web3jService, DiscardResponse.class);
   }
 
-  public Request<?, ProposalsResponse> ibftProposals() {
+  public Request<?, ProposalsResponse> proposals() {
```

Comment:
probably should be called getProposals given the name of the functions below.

Reply:
You're right, these should be consistent